### ISSUE TITLE
Artists Missing from MySQL

### ISSUE THREAD

**Issue by synixebrett** [SaskArtists/SaskArtists#254]
Some artists are missing in MySQL, mainly the artists that were not in GitHub. We need to add them in before this year is over.

**Comment by rye761** [comment]
I can do this, but only after the new artists are present on the server.

**Comment by BmanDesignsCanada** [comment]
They are present on the server, they have been added to git and the server has pulled.

**Comment by gemohandy** [comment]
Making a list of artists who are on the server, but don't have database positions:

Improperly Named:
- [ ] Anthony K. Linklater - listed in server as anthony, not alinklater
- [ ] Inglis Sheldon Williams - listed in server as inglis, not iwilliams

Properly Named: (First letter of first name, then last name)
- [ ] Buffy Sainte-Marie
- [ ] David Loran
- [ ] David Ross
- [ ] Huang Zhong Yang 
- [ ] Jack Cowin 
- [ ] Micheal Sereda 
- [ ] Wilf Perreault

Also, please add descriptions, so that these artists are not added to the list in #182

**Comment by synixebrett** [comment]
Closing as we will be moving to the artist.json system.

I will be creating a new issue for the improperly named artists.
